jasper_reports/JasperReports/abstract_data_generator.py

```diff
@@ -31,6 +31,10 @@
 
 
 class AbstractDataGenerator:
+
+    def __init__(self):
+        pass
+
     # Simple function all DataGenerators should implement
-    def generate(self, fileName):
+    def generate(self, filename):
```

Comment:
It is override in subclasses. 
I not add this code, only migrate to Odoo 10.0 and adjust PEP8 in python code